### PR TITLE
Luw 51 Parallax section

### PR DESCRIPTION
**Pull request by fannymartensson**

### CODE REVIEW

File: src/assets/data/data.json [hunk 1]
```diff
@@ -191,5 +202,119 @@
...
+            },
+            { "id": 2, "text": "Personligt anpassad grafisk profil." },
+            {
+              "id": 3,
+              "text": "Designsystem"
+            },
+            { "id": 4, "text": "Designstrategi" },
+            {
+              "id": 5,
+              "text": "Användaranays"
```
Comment (olivernygren) [inline src/assets/data/data.json:313]: Spelling error => Designana(l)ys

**Review by olivernygren — APPROVED** [review]

File: src/components/Sections/Service/Service.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Parallax } from './Parallax';
+
+export const Service = ({ data }: any) => {
```
Comment (boomboxbacardi) [inline src/components/Sections/Service/Service.tsx:3]: Can we type the service interface?

File: src/pages/ServicePage/ServicePage.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  Navbar,
+  Service,
+} from '../../components/Sections';
+
+export const ServicePage = () => {
+  const { slug } = useParams();
+  const showPopup = useStore((state) => state.showPopup);
+  const [slugData, setSlugData] = useState() as any;
+
+  const mappedArray = data.servicePage.map((x: any) => x);
```
Comment (boomboxbacardi) [inline src/pages/ServicePage/ServicePage.tsx:19]: mappedArray
Can we use another variable name for clarification?
Also, (x: any) => x) make this more descriptive

File: src/pages/ServicePage/ServicePage.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+  useEffect(() => {
+    mappedArray.forEach((o: any) => {
```
Comment (boomboxbacardi) [inline src/pages/ServicePage/ServicePage.tsx:22]: (o: any)
Make more descriptive and type it out

**Review by boomboxbacardi — APPROVED** [review]
Smaller fixes before you merge!

File: src/components/Sections/Service/Service.tsx (hunk 2 above)
Comment (fannymartensson) [reply src/components/Sections/Service/Service.tsx:3]: Yes I will add this.

File: src/pages/ServicePage/ServicePage.tsx (hunk 3 above)
Comment (fannymartensson) [reply src/pages/ServicePage/ServicePage.tsx:19]: Yes I can clarify this.

**Pull request merged**